test/functional/wallet_multiwallet.py

```diff
@@ -231,8 +233,14 @@ def run_test(self):
         assert_equal(loadwallet_name['name'], new_wallet_name)
         w10 = node.get_wallet_rpc(new_wallet_name)
         assert_equal(w10.getwalletinfo()['walletname'], new_wallet_name)
-
         assert new_wallet_name in self.nodes[0].listwallets()
 
+        self.log.info("Test disableprivatekeys creation.")
```

Comment:
It's not obvious to me why this test is in `wallet_multiwallet.py`. What does it have to do with multiwallet?

Reply:
I guess I placed it there because this was the only test that used `createwallet`. But right, I shall move it to a new test.